sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/IPeeringManagementClient.cs

```diff
@@ -142,19 +147,17 @@ public partial interface IPeeringManagementClient : System.IDisposable
         /// Checks if the peering service provider is present within 1000 miles
         /// of customer's location
         /// </summary>
-        /// <param name='peeringServiceLocation'>
-        /// Gets or sets the peering service location.
-        /// </param>
-        /// <param name='peeringServiceProvider'>
-        /// Gets or sets the peering service provider.
+        /// <param name='checkServiceProviderAvailabilityInput'>
+        /// The CheckServiceProviderAvailabilityInput indicating customer
+        /// location and service provider.
         /// </param>
         /// <param name='customHeaders'>
         /// The headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        Task<AzureOperationResponse<string>> CheckServiceProviderAvailabilityWithHttpMessagesAsync(string peeringServiceLocation = default(string), string peeringServiceProvider = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<string>> CheckServiceProviderAvailabilityWithHttpMessagesAsync(CheckServiceProviderAvailabilityInput checkServiceProviderAvailabilityInput, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This is a breaking change. Please let us know if we're missing something. Thanks.

Reply:
There was no change in the rest api spec for this and used the same autorest cmd to generate the SDK as before. Am I missing something here ?